### ISSUE TITLE
way to add sdz (or ideally, a whole folder of sdzs) onto rapid without repositories

### ISSUE THREAD

**Issue by nicfreeman1209** [spring/RapidTools#48]

**Comment by abma** [comment]
which tags should be used? example?

**Comment by nicfreeman1209** [comment]
for example i would make a folder containing all the sdz from http://imolarpg.dyndns.org/trac/balatest/browser/etc/missions/campaign/beta%20release%20sdzs which i could download from rapid using the tag barm:test and barm:stable

**Comment by nicfreeman1209** [comment]
if i could also include maps in batch, as part of the same folder, that would be ideal

**Comment by zwzsg** [comment]
It used to exists: https://springrts.com/phpbb/viewtopic.php?t=22077

By login to http://packages.springrts.com/ I used to be able to add new version of kp to rapid without having a kp repository.

But today I couldn't get it to work.
I fill the field, then I click upload. It does the uploading for a while, and then I am returned to the login page, and I don't see any new entry in http://repos.springrts.com/versions.gz

**Comment by abma** [comment]
> But today I couldn't get it to work.
> I fill the field, then I click upload. It does the uploading for a while, and then I am returned to the login page, and I don't see any new entry in http://repos.springrts.com/versions.gz

no clue whats broken. which version of KP did you try to upload (link to .sdz please) / which tag did you use?

does there a git repository exist of KP?

**Comment by zwzsg** [comment]
Woah, wasn't actually expecting an answer! (o_o)

- I tried to upload [Kernel Panic 4.7](https://springrts.com/phpbb/viewtopic.php?f=43&t=36267)
- Here is its sd7: http://springfiles.com/spring/games/kernel-panic-mod
- Before uploading to rapid, I un7zip it and rezip it as .sdz*
- I used 4.7 as version, and stable as tag
- There is no git nor even svn for KP.

\* It's how I was told to do, how I did it for previous versions, and somehow sd7 and .sdz can play together without sync issue.
